tests/test_ml-notebook.py

```diff
@@ -49,7 +49,7 @@ def test_jax_random_number_generator():
     with jax.default_device(jax.devices("cpu")[0]):
         key = random.key(seed=42)
         x = random.normal(key=key)
-        np.testing.assert_allclose(x, -0.18471177)
+        np.testing.assert_allclose(x, np.float32(-0.028305))
```

Comment:
Sigh, this still needs more decimal places :upside_down_face:

```
>           np.testing.assert_allclose(x, np.float32(-0.028305))
E           AssertionError: 
E           Not equal to tolerance rtol=1e-07, atol=0
E           
E           Mismatched elements: 1 / 1 (100%)
E           Max absolute difference among violations: 3.837049e-07
E           Max relative difference among violations: 1.3556082e-05
E            ACTUAL: array(-0.028305, dtype=float32)
E            DESIRED: array(-0.028305, dtype=float32)
```

I'll do an install locally and work this one out.

Reply:
```suggestion
        np.testing.assert_allclose(x, -0.028304616)
```

Oh, and L60 needs to change too, I'll push a commit.